de-DE.Rmd

```diff
@@ -1,5 +1,97 @@
 ---
-title: "`r emo::ji('Germany')` Project-wide Code of Conduct statement for Bioconductor (German, Germany)"
+title: "`r emo::ji('Germany')` Projektweite Erklärung zum Verhaltenskodex von Bioconductor (Deutsch, Deutschland)"
 ---
 
-TODO.
+(Adaptiert vom [BioC 2020 Verhaltenskodex][1])
```

Comment:
Can you replace this line by the translation of 
> (Adapted from the [Code of conduct for the conference BioC 2020][1], translated from [American English](en-US.html))

See for an example, in French
https://github.com/Bioconductor/bioc_coc_multilingual/blob/main/fr-FR.Rmd#L5

Reply:
done.